src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,88 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+type Props = {
+  onAdd: (movie: Movie) => void,
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [valueTitle, setValue] = useState('');
+  const [valueDescription, setValueDescription] = useState('');
+  const [valueImageUrl, setValueImageUrl] = useState('');
+  const [valueImdbUrl, setValueImdbUrl] = useState('');
+  const [valueImdbId, setValueImdbId] = useState('');
+
+  const checkButton
+    = valueTitle !== ''
+    && valueImageUrl !== ''
+    && valueImdbUrl !== ''
+    && valueImdbId !== '';
```

Comment:
```suggestion
  const areValuesEntered = valueTitle !== ''
    && valueImageUrl !== ''
    && valueImdbUrl !== ''
    && valueImdbId !== '';
```
Fix formatting. Also, it's better to name boolean variables according to these rules, so it's clear that it is a boolean value
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig